Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Graphs"
 uuid = "86223c79-3864-5bf0-83f7-82e725a168b6"
-version = "1.9.0"
+version = "1.9.1"
```

Comment:
actually it's a new feature so semver says 1.10

Reply:
Okay 👍